src/packages/cli/package.json

```diff
@@ -68,23 +68,23 @@
     "@prisma/introspection": "workspace:*",
     "@prisma/migrate": "workspace:*",
     "@prisma/sdk": "workspace:*",
-    "@prisma/studio": "0.236.0",
-    "@prisma/studio-server": "0.236.0",
-    "@prisma/studio-transports": "0.236.0",
-    "@prisma/studio-types": "0.236.0",
+    "@prisma/studio": "0.238.0",
+    "@prisma/studio-server": "0.238.0",
+    "@prisma/studio-transports": "0.238.0",
+    "@prisma/studio-types": "0.238.0",
```

Comment:
@madebysid Is this normal that this is done via renovate and not triggered by you?

Reply:
Yeah should be okay. Usually it happens automatically, but because Buildkite was broken for a bit, that failed